tests/test_admin.py

```diff
@@ -23,6 +28,42 @@ def setUp(self):
         self.normaluser.save()
         self.options = admin.site._registry[self.model]
 
+    # @mock.patch("constance.admin.ConstanceForm.clean_version", lambda x: 'test')
```

Comment:
Why this is commented?

Reply:
We should not have any commented code, if is not needed then remove it.

Reply:
remove it!